src/app/components/header/BBCNewsBanner.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const BBCNewsLink = styled.a`
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  font-size: 2.2em;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+`;
+
+const BBCNewsHeader = styled.header`
+  background-color: #bb1919;
+  height: 40px;
+  width: 100%;
+  padding: 15px 16px;
+`;
+
+class BBCNewsBanner extends Component {
```

Comment:
This doesn't need to be a stateful React component and can be a [stateless component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components).

Can then hardcode the title rather than fetching it from state.

Finally, this component should be called `Header`, not `BBCNewsBanner`.

Reply:
Ok. Sareh suggested I changed it from Header to BBCNewsBanner

Reply:
Sorry for the confusion, Jonathan - I meant renaming variables within the file, rather than the component itself. I'm sorry that my comment wasn't clear. I'll make sure to be more precise in future.